mwt-setup-scripts/configure-cluster-for-jenkins.sh

```diff
@@ -0,0 +1,366 @@
+	dcos security org users grant edge-lb-principal dcos:mesos:master:framework:principal:edge-lb-principal full
+	dcos security org users grant edge-lb-principal dcos:mesos:master:framework:role full
+	dcos security org users grant edge-lb-principal dcos:mesos:master:reservation:principal:edge-lb-principal full
+	dcos security org users grant edge-lb-principal dcos:mesos:master:reservation:role full
+	dcos security org users grant edge-lb-principal dcos:mesos:master:volume:principal:edge-lb-principal full
+	dcos security org users grant edge-lb-principal dcos:mesos:master:volume:role full
+	dcos security org users grant edge-lb-principal dcos:mesos:master:task:user:root full
+	dcos security org users grant edge-lb-principal dcos:mesos:master:task:app_id full
+	
+cat >$DIR/edge-lb-options.json <<EOF
+{
+  "service": {
+    "secretName": "dcos-edgelb/edge-lb-secret",
+    "principal": "edge-lb-principal",
+    "mesosProtocol": "https"
+  }
+}
+EOF
+
+	dcos package install --options=$DIR/edge-lb-options.json edgelb --yes
+	until dcos edgelb ping; do sleep 1; done
+
+	echo "EdgeLB installed!"
+}
+
+install_edgelb_jenkins_pools()
+{
+    dcos package repo add --index=0 edgelb-pool  "${EDGELB_POOL_STUB_URL}"
+
+cat >$DIR/jenkins-pool-configuration.json <<EOF
+{
+  "apiVersion": "V2",
+  "name": "jenkins-pool",
+  "count": 1,
+  "haproxy": {
+    "frontends": [
+      {
+        "bindPort": 80,
+        "protocol": "HTTP",
+        "linkBackend": {
+          "map": [
+            {
+              "pathBeg": "/jenkins1",
+              "backend": "backend-jenkins0"
+            },
+            {
+              "pathBeg": "/jenkins2",
+              "backend": "backend-jenkins1"
+            }
+          ]
+        }
+      }
+    ],
+    "backends": [
+      {
+        "name": "backend-jenkins0",
+        "protocol": "HTTP",
+        "services": [
+          {
+            "mesos": {
+              "frameworkName": "mom-4",
+              "taskName": "jenkins0"
+            },
+            "endpoint": {
+              "portName": "nginx",
+              "type": "AGENT_IP"
+            }
+          }
+        ]
+      },
+      {
+        "name": "backend-jenkins1",
+        "protocol": "HTTP",
+        "services": [
+          {
+            "mesos": {
+              "frameworkName": "mom-4",
+              "taskName": "jenkins1"
+            },
+            "endpoint": {
+              "portName": "nginx",
+              "type": "AGENT_IP"
+            }
+          }
+        ]
+      }
+    ]
+  }
+}
+EOF
+
+	dcos security org users grant edge-lb-principal dcos:adminrouter:service:dcos-edgelb/pools/jenkins-pool full
+
+	# Create jenkins pool	
+	dcos edgelb create $DIR/jenkins-pool-configuration.json
+	
+	# Verify
+	dcos edgelb endpoints jenkins-pool
+}
+
+install_edgelb_simplehttp_pools()
+{
+    dcos package repo add --index=0 edgelb-pool  "${EDGELB_POOL_STUB_URL}"
+
+cat >$DIR/simple-http-configuration.json <<EOF
+{
+  "apiVersion": "V2",
```

Comment:
Normally, pool servers try to use port 9090 for stats port, which prevents having more than one pool server per host. I would recommend setting in the `haproxy` section the following:
```
      "stats": {
         "bindPort": <unique-per-pool-port>
      },
```
and having each ELB pointing to Edge use this port for healtchecking.

It's also possible to use newly-released AWS integration feature - this would make Edge configure AWS for us,but this requires some extra legwork wrt. permissions.